ntbs-service/Pages/Notifications/LegacyView.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@page "/Notifications/LegacyView/{LegacyNotificationId}"
```

Comment:
Definitely is a different journey to the other notification pages but think it might still be worth keeping to the "/Notifications/{LegacyNotificationId}/LegacyView" page route convention

Reply:
It will be a bit confusing to have legacyNotification id after notification as it will look lile notification id

Reply:
yes, this look better :)